### ISSUE TITLE
Support for @ operator ?

### ISSUE THREAD

**Issue by eserie** [jonasrauber/eagerpy#35]
It could be great to have support for the @ operator.

Given two tensors:
```python
a = ep.astensor(np.random.normal(size=(2, 10)))
b = ep.astensor(np.random.normal(size=(10, 3)))
```

The expression:
```python
c = a.matmul(b)
```

could be written as:
```python
c = a @ b
```

**Comment by jonasrauber** [comment]
Agree. I think all we have to do is to overwrite `__matmul__` to delegate to `matmul`.

**Comment by eserie** [comment]
Ok great!
Do you want I propose a PR for that?

**Comment by jonasrauber** [comment]
Feel free to do so.

**Comment by eserie** [comment]
I created the PR https://github.com/jonasrauber/eagerpy/pull/37#.

**Comment by jonasrauber** [comment]
merged #37, so closing this :)

**Comment by eserie** [comment]
Thanks!
